### PR TITLE
 takethetime: fetch from PyPI and disable checks

### PR DESCRIPTION
**Pull request by huantianad**
Previously, we built the package from GitHub in order to run the included tests. However, because those tests are timing dependent, they couldpotentially fail even when the package itself was fine. Because the tests were not useful, we switched back over to getting the package from PyPI and only using `pythonImportsCheck`.
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by SuperSandro2000 — COMMENTED** [review]
Can you create git tags we can use instead of commits?

**Comment by huantianad** [comment]
Done ✅

**Comment by SuperSandro2000** [comment]
@ofborg build python310Packages.takethetime python311Packages.takethetime

File: pkgs/development/python-modules/takethetime/default.nix [hunk 1]
```diff
@@ -25,13 +25,6 @@ buildPythonPackage {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/takethetime/default.nix:25]: Please disable test_average and test_duration since they seem to be timing sensitive

Comment (huantianad) [reply pkgs/development/python-modules/takethetime/default.nix:25]: Those are the only two tests, should we remove `pytestCheckHook` entirely and rely on `pythonImportsCheck`?

Comment (SuperSandro2000) [reply pkgs/development/python-modules/takethetime/default.nix:25]: Sure, if there are no other tests. I don't think there is any advantage to fetch from pypi.

Comment (SuperSandro2000) [reply pkgs/development/python-modules/takethetime/default.nix:25]: Sure, if there are no other tests. I don't think there is any advantage to fetch from pypi.

**Comment by huantianad** [comment]
@ofborg  build python310Packages.takethetime python311Packages.takethetime python310Packages.aw-core python311Packages.aw-core

File: pkgs/development/python-modules/takethetime/default.nix [hunk 2]
```diff
@@ -1,37 +1,26 @@
...
+    inherit version;
+    sha256 = "sha256-2+MEU6G1lqOPni4/qOGtxa8tv2RsoIN61cIFmhb+L/k=";
   };
 
   disabled = pythonOlder "3.6";
 
-  nativeCheckInputs = [ pytestCheckHook ];
-
-  pytestFlagsArray = [ "tests/tests.py" ];
+  # Tests are timing dependent and aren't included with PyPI distribution
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/takethetime/default.nix:19]:
```suggestion
  # all tests are timing dependent
```

**Pull request merged**